hyperspy/drawing/mpl_he.py

```diff
@@ -96,6 +96,15 @@ def plot_navigator(self):
         elif len(self.navigator_data_function().shape) >= 2:
             imf = image.ImagePlot()
             imf.data_function = self.navigator_data_function
+            imf.colorbar = kwargs.get('colorbar', True)
```

Comment:
Adding a feature? Then report it in the header and the User Guide.

Wouldn't it be better to have a `navigator_kwds` argument?